app/controllers/groups_controller.rb

```diff
@@ -85,7 +85,7 @@ def show
 
     @discussions = @discussions.joined_to_current_motion.
                                 preload(:current_motion, {:group => :parent}).
-                                order('motions.closing_at ASC, last_activity_at DESC').
+                                order('motions.closing_at ASC, last_item_at, discussions.created_at DESC').
```

Comment:
this does not do what I think it does.. and I think we need a condition statement